js/build/field-attachment.js

```diff
@@ -213,8 +213,21 @@ var ShortcodeAttribute = Backbone.Model.extend({
 		label:       '',
 		type:        '',
 		value:       '',
-		placeholder: '',
+		description: '',
+		customAttributes: {
+			placeholder: '',
+		}
 	},
+
+	initialize: function() {
+
+		// Handle legacy custom attributes.
+		if ( this.get('placeholder' ) ) {
```

Comment:
> defined in one place

Not sure what you mean here. I've moved alongside the other code that handles these custom attributes. 

Ticket for removal: #292